README.md

```diff
@@ -0,0 +1,24 @@
+![Banner](https://media.licdn.com/dms/image/D5616AQGqULKfQUchEQ/profile-displaybackgroundimage-shrink_350_1400/0/1697104932324?e=1705536000&v=beta&t=VU7h3Z5gl5-9BueHGVp8odgRJRQxJraDRH50g_Y4jL0)
+
+[<img align="left" height="100" src="https://avatars.githubusercontent.com/u/145749994?s=200&v=4">](https://avatars.githubusercontent.com/u/145749994?s=200&v=4)
+
+## Welcome to tapaScript with Tapas Adhikary: Unlocking the Art of Code Mastery 🚀
+
+Greetings, Developers! I am Tapas Adhikary, your guide on this exhilarating journey into the realms of UI, UX, Web Development, JavaScript, React, Next.js, Node.js, and the captivating universe of FullStack engineering. Having immersed myself in the world of code since 2000, I bring 19 years of experience in crafting production-ready software.
+
+At tapaScript, my commitment is not just to teach but to make the fundamentals an unforgettable part of your coding repertoire. Picture mastering programming with unwavering confidence - that's the essence of my approach.
+
+My coding odyssey, initiated in 2004, has been a remarkable adventure. From Open Source contributions, notably in ReactPlay, to my engagement with various platforms such as "GreenRoots Blog," FreeCodeCamp, CSS-Tricks, and more, I have honed my skills and shared my insights.
+
+Join me on YouTube, where I pour years of gathered wisdom into empowering you. Together, let's make tapaScript a place where fundamental concepts become second nature, and you emerge with the confidence to navigate the intricacies of software development.
+
+Welcome to tapaScript - where the beauty of mastering code unfolds. 👨‍💻✨
```

Comment:
Can we centre this section?

![image](https://github.com/tapascript/.github/assets/3633137/25e65d3f-fc84-43a1-bec6-14ff87b09d83)

Reply:
Also remove the Gmail... you can add FB if available :)

Reply:
- [x] Centered that section
- [x] Removed Gmail
- [x] Added FB and WP
- [x] Added YouTube Badge